Configuration/Development.php

```diff
@@ -2,12 +2,7 @@
 
 // Disable frontend caching (single hit caching)
 $configureCache = function($name) {
-    // TYPO3 v9
-    $index = 'cache_' . $name;
-    if (!isset($GLOBALS['TYPO3_CONF_VARS']['SYS']['caching']['cacheConfigurations'][$index])) {
-        // TYPO3 v10 and later
-        $index = $name;
-    }
+    $index = $name;
```

Comment:
@denislorch, kann man sich glaub sparen, das in ne extra Variable zu kopieren ;-)